Tests/Data/Mechanics/Linear/square_with_deactivated_hole.prj

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<OpenGeoSysProject>
+    <mesh>square_with_2_matIDs.vtu</mesh>
+    <geometry>disc_with_hole.gml</geometry>
+    <processes>
+        <process>
+            <name>SD</name>
+            <type>SMALL_DEFORMATION</type>
+            <integration_order>2</integration_order>
+            <constitutive_relation id="0">
+                <type>LinearElasticIsotropic</type>
+                <youngs_modulus>E</youngs_modulus>
+                <poissons_ratio>nu</poissons_ratio>
+            </constitutive_relation>
+            <constitutive_relation id="1">
```

Comment:
If id=1 is deactivated, there is no need to have the constitutive relation, I'd say.

Reply:
removed.